Tests/web3swiftTests/localTests/EthereumAddressTest.swift

```diff
@@ -35,4 +35,15 @@ class EthereumAddressTest: XCTestCase {
         XCTAssertNil(ethereumAddress)
     }
 
+    func testDescription() async throws {
+        let rawAddress = "0x200eb5ccda1c35b0f5bf82552fdd65a8aee98e79"
+        let ethereumAddress = EthereumAddress(rawAddress)
+
+        let sut = String(describing: ethereumAddress)
+        print(sut)
```

Comment:
Please, don't leave `print` in the code. Some time ago we decided not to use it.
Only for debugging purposes.

Reply:
Apologies, my bad!